extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/basePage.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { WizardPageBase } from '../../wizardPageBase';
+import { DeployAzureSQLDBWizard } from '../deployAzureSQLDBWizard';
+
+export abstract class BasePage extends WizardPageBase<DeployAzureSQLDBWizard> {
+
+	protected liveValidation!: boolean;
+
+	public initialize(): void {
+		throw new Error('Method not implemented.');
+	}
+
+	protected async validatePage(): Promise<string> {
+		return '';
+	}
+
+	protected activateRealTimeFormValidation(): void {
```

Comment:
This doesn't seem to be used anywhere? And I'm not even sure what this is supposed to be doing. If it's not used please remove for now.

Reply:
it's used in database settings page to call validate page in real time.

Reply:
But why is this necessary? ModelView forms will handle the validation automatically if you set the appropriate properties. This shouldn't be necessary.

If you don't want validation on a field then you shouldn't be setting those properties.

Reply:
A lot of our other wizards/dialogs do this - for example the MIAA deployment. 

(I assume the below is what you mean by "realtime validation". If that isn't please explain in detail what behavior you want to happen here)

![Capture](https://user-images.githubusercontent.com/28519865/95636714-a8bdd880-0a44-11eb-9b5e-e75ed4b4df5f.gif)